dist/react-datepicker.js

```diff
@@ -270,9 +272,9 @@ return /******/ (function(modules) { // webpackBootstrap
 	      todayButton: this.props.todayButton,
 	      outsideClickIgnoreClass: outsideClickIgnoreClass,
 	      timezone: this.props.timezone,
-	      timePickerButton: this.props.timePickerButton,
+	      timePickerButton: this.props.timeDisabled ? false : this.props.timePickerButton,
 	      onToggle: this.handleToggleTime,
-	      showTimePicker: this.state.showTimePicker,
+	      showTimePicker: this.props.timeDisabled ? false : this.state.showTimePicker,
```

Comment:
Since this is `getInitialState`, can you include the default value for `this.state.showTimePicker` explicitly here? Although it looks like the code was like that.